testcontainers/neo4j.py

```diff
@@ -25,13 +25,15 @@ class Neo4jContainer(DbContainer):
 
     Example
     -------
-    ::
-        with Neo4jContainer() as neo4j:
-            with neo4j.get_driver() as driver:
-                with driver.session() as session:
-                    result = session.run("MATCH (n) RETURN n LIMIT 1")
-                    record = result.single()
+    .. doctest::
 
+        >>> from testcontainers.neo4j import Neo4jContainer
+
+        >>> with Neo4jContainer() as neo4j:
```

Comment:
Could use `with ..., ..., ...` here to keep the indents under control.

Reply:
resolved